code/plugins/api/users/users/config.php

```
@@ -0,0 +1,234 @@
+
+		$app   = Factory::getApplication();
+		$cdata = array();
+
+		$xmlFile = JPATH_ADMINISTRATOR . '/components/' . $cname . '/' . $name . '.xml';
+		if (file_exists($xmlFile)) {
+			$xml = simplexml_load_file($xmlFile);
+			$cdata['version'] = (string) $xml->version;
+		} else {
+			$cdata['version'] = 'Unknown';
+		}
+		$jconfig          = Factory::getConfig();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$jconfig'. (Unused Code Rules)

(UnusedLocalVariable)